modules/gapi/include/opencv2/gapi/opencv_includes.hpp

```diff
@@ -14,6 +14,12 @@
 #  include <opencv2/core/cvdef.h>
 #  include <opencv2/core/types.hpp>
 #  include <opencv2/core/base.hpp>
+#define GAPI_OWN_TYPES_LIST     cv::gapi::own::Rect,                           \
+                                cv::gapi::own::Size,                           \
+                                cv::gapi::own::Point,                          \
+                                cv::gapi::own::Point2f,                        \
+                                cv::gapi::own::Scalar,                         \
+                                cv::gapi::own::Mat
```

Comment:
Why use macro here and not an enumeration via tuple?

Macros seem old-fashioned (and sometimes dangerous) now when we have variadic templates